mariadb/kontena.yml

```diff
@@ -0,0 +1,83 @@
+stack: kontena/mariadb
+version: 0.2.0
+expose: server
+variables:
+  root_password:
+    type: string
+    from:
+      vault: ${STACK}-root-password
+      random_string: 16
+    to:
+      vault: ${STACK}-root-password
+  user:
+    type: string
+    default: mariadb
+    from:
+      vault: ${STACK}-user
+      prompt: Username
+    to:
+      vault: ${STACK}-user
+  password:
+    type: string
+    from:
+      vault: ${STACK}-password
+      random_string: 16
+    to:
+      vault: ${STACK}-password
+  db:
+    type: string
+    default: mariadb
+    from:
+      vault: ${STACK}-database
+      prompt: Database name
+    to:
+      vault: ${STACK}-database
+  affinity:
+    type: string
+    default: label==~mariadb
+    from:
+      prompt: Affinity
+  mem_limit:
+    type: integer
+    default: 512
+    from:
+      prompt: Memory limit (MB)
+  innodb_buffer_pool_size:
+    type: string
+    from:
+      evaluate: ((${mem_limit} * 10) * 7) / 100
```

Comment:
Why such a complex formula? Is there some reason why `${mem_limit} * 0,7` would not work?

Reply:
It needs to be an integer.